src/pages/workspaces/workspace/ShareWorkspaceModal/Collaborator.ts

```diff
@@ -0,0 +1,156 @@
+import _ from 'lodash/fp';
+import React, { Dispatch, SetStateAction } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { LabeledCheckbox, Link, Select, SelectProps } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import { getPopupRoot } from 'src/components/popup-utils';
+import colors from 'src/libs/colors';
+import { getUser } from 'src/libs/state';
+import * as Utils from 'src/libs/utils';
+import { BaseWorkspace, isAzureWorkspace, WorkspaceAccessLevel } from 'src/libs/workspace-utils';
+import { AccessEntry, WorkspaceAcl } from 'src/pages/workspaces/workspace/WorkspaceAcl';
+
+interface CollaboratorProps {
+  aclItem: AccessEntry; // the item to render
+  acl: WorkspaceAcl; // the entire current acl
+  setAcl: Dispatch<SetStateAction<WorkspaceAcl>>;
+  originalAcl: WorkspaceAcl; // the original acl, to determine new items
+  workspace: BaseWorkspace; // the workspace the acl belongs to
+  lastAddedEmail?: string; // the most recently added email to the list
+}
+
+/**
+ *  Note: the select menu of each Collaborator item NOT focusable via keyboard
+ *  tab will only focus on the remove buttons
```

Comment:
I'm not sure what you mean by the select menu not being keyboard focusable. I am to use it by opening it with the Space key and using up and down to traverse. The focus indication around it is incredibly subtle though.

<img width="549" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/d2dca754-7fbe-4990-bb16-8168918523b3">

Reply:
I didn't even think to hit the space button, because it looked like the focus was only on the remove button, so I thought it would remove the user.
I stand corrected.